lib/matplotlib/axes/_axes.py

```diff
@@ -3069,9 +3069,9 @@ def stem(self, *args, linefmt=None, markerfmt=None, basefmt=None, bottom=0,
 
     @_preprocess_data(replace_names=["x", "explode", "labels", "colors"])
     def pie(self, x, explode=None, labels=None, colors=None,
-            autopct=None, pctdistance=0.6, shadow=False, labeldistance=1.1,
-            startangle=0, radius=1, counterclock=True,
-            wedgeprops=None, textprops=None, center=(0, 0),
+            autopct=None, wedgelabels=None, wedgelabelsfmt=None, pctdistance=0.6,
+            shadow=False, labeldistance=1.1, startangle=0, radius=1,
+            counterclock=True, wedgeprops=None, textprops=None, center=(0, 0),
             frame=False, rotatelabels=False, *, normalize=True, hatch=None):
```

Comment:
The new arguments must be added after the `*` in case someone is using positional arguments.